tests/models/mpt/test_modeling_mpt.py

```diff
@@ -498,15 +500,21 @@ def test_generation_batched(self):
 
         inputs = tokenizer(input_texts, return_tensors="pt", padding=True).to(torch_device)
 
-        expected_output = [
-            "Hello my name is Tiffany and I am a mother of two beautiful children. I have been a nanny for the",
-            "Today I am going at the gym and then I am going to go to the grocery store. I am going to buy some food and some",
-        ]
+        expected_output = {
+            "cuda": [
+                "Hello my name is Tiffany and I am a mother of two beautiful children. I have been a nanny for the",
+                "Today I am going at the gym and then I am going to go to the grocery store. I am going to buy some food and some",
+            ],
+            "xpu": [
+                "Hello my name is Tiffany. I am a mother of two beautiful children. I have been a nanny for over",
+                "Today I am going at the gym and then I am going to go to the mall with my mom. I am going to go to the",
+            ],
+        }
```

Comment:
@jiqing-feng , pls use `Expectations` to set device specific expectations, you can refer to https://github.com/huggingface/transformers/pull/37644, thx

Reply:
done.